instrumentation/opentelemetry-instrumentation-requests/tests/test_requests_integration.py

```diff
@@ -335,6 +355,27 @@ def test_requests_timeout_exception(self, *_, **__):
         span = self.assert_span()
         self.assertEqual(span.status.status_code, StatusCode.ERROR)
 
+    def test_adapter_with_custom_response(self):
+        # See ONE-56722
```

Comment:
Hm, what is this? :thinking:

Reply:
thx, i removed it (just some ticket number)